tox.ini

```
@@ -4,7 +4,7 @@
 # directory.
 
 [tox]
-envlist = py27, py35, py36, py37, py38, py39, pypy
+envlist = py27, py35, py36, py37, py38, py39, py310, py311, pypy
```

Add `py312` and refactor:
```suggestion
envlist = py{27, 35, 36, 37, 38, 39, 310, 311, 312, py}
```

---

Done in https://github.com/xolox/python-humanfriendly/pull/69/commits/11d62fbb1548cd4280cb4bdecae5d70910f8910d